docs/devices/SLR2.md

```diff
@@ -28,17 +39,33 @@ description: "Integrate your Hive SLR2 via Zigbee2MQTT with whatever smart home
+- `running_state`: The current running state. Read-only. Possible states are: `idle`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/heat/get` with payload `{"running_state": ""}`.
 
 ### Climate (water endpoint)
-This climate device supports the following features: `occupied_heating_setpoint`, `local_temperature`, `system_mode`, `running_state`, `pi_heating_demand`.
-- `occupied_heating_setpoint`: Temperature setpoint. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"occupied_heating_setpoint_water": VALUE}` where `VALUE` is the °C between `7` and `30`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"occupied_heating_setpoint_water": ""}`.
-- `local_temperature`: Current temperature measured on the device (in °C). To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"local_temperature_water": ""}`.
-- `system_mode`: Mode of this device. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"system_mode_water": VALUE}` where `VALUE` is one of: `off`, `auto`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"system_mode_water": ""}`.
-- `running_state`: The current running state. Possible values are: `idle`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"running_state_water": ""}`.
+This climate device supports the following features: `temperature_setpoint_hold`, `temperature_setpoint_hold_duration`, `system_mode`, `running_state`, `pi_heating_demand`.
+- `temperature_setpoint_hold`: Used to hold (`false`) or release (`true`) the hotwater relay. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/water/set` with payload `{temperature_setpoint_hold": VALUE}` where `VALUE` is `true` or  `false`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/water/get` with payload `{"temperature_setpoint_hold": ""}`.
+- `temperature_setpoint_hold_duration`: Temperature setpoint hold duration (emergency_heating only) - used to determine how long the emergency heating period isrelay. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/water/set` with payload `{temperature_setpoint_hold_duration": VALUE}` where `VALUE` is number of minutes. There is no read command for this value (i.e. write-only).
+- `system_mode`: Mode of this device. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/water/set` with payload `{"system_mode": VALUE}` where `VALUE` is one of: `off`, `auto`, `heat`,`emergency_heating` . To read send a message to `zigbee2mqtt/FRIENDLY_NAME/water/get` with payload `{"system_mode": ""}`.
+- `running_state`: The current running state. Read-only. Possible values are: `idle`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/water/get` with payload `{"running_state": ""}`.
+
+To switch water ON send the following messages / payloads:
```

Comment:
Please don't make manual changes in this part of the documentation, they are auto generated based on the `exposes`. You can move it under the pairing section.

Reply:
also the auto-generated values for `system_mode` are missing `emergency_heating`  for example